samples/DotNet/Azure.Messaging.EventHubs/ManagedIdentityWebApp/SendReceive.aspx.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using Azure.Messaging.EventHubs;
+using Azure.Identity;
+using System.Threading.Tasks;
+
+// Always add app to IAM roles
+// Don't use on deployment slots but only on root
+namespace ManagedIdentityWebApp
+{
+    public partial class SendReceive : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        protected async void btnSend_Click(object sender, EventArgs e)
+        {
+            await using (EventHubProducerClient producerClient = new EventHubProducerClient($"{txtNamespace.Text}.servicebus.windows.net", txtEventHub.Text, new DefaultAzureCredential()))
```

Comment:
Not sure if it's a concern for you, but this URL is potentially unstable across different clouds.   For example, this pattern  is unlikely to work with an Azure subscription in the US Government cloud or the China cloud.

Since it now appears in the portal as part of the namespace properties (albeit still not so clear), you may want to consider having the full value provided.

![image](https://user-images.githubusercontent.com/913445/72082099-a53acc80-32cd-11ea-8c1b-14058d62cf53.png)

Reply:
Taking the fully qualified namespace now as the input.